extensions/warp-ipfs/src/thumbnail.rs

```diff
@@ -1,12 +1,17 @@
 use bytes::Bytes;
 use futures::channel::oneshot;
-use image::{ImageFormat, ImageReader};
+use image::{
+    codecs::gif::{GifDecoder, GifEncoder, Repeat},
+    AnimationDecoder, DynamicImage, Frame, ImageFormat, ImageReader,
+};
 use rust_ipfs::{Ipfs, IpfsPath};
 use std::{
     collections::BTreeMap,
     ffi::OsStr,
     fmt::Display,
+    fs::File,
```

Comment:
It would be best to move `use std::fs::File` into the blocking task scope within `ThumbnailGenerator::insert` since it does not support wasm32 target. Would make it easier so we dont have to gate it behind a feature